repo/packages/F/flink/2/config.json

```diff
@@ -0,0 +1,286 @@
+           "default":""
+        },
+        "secret":{
+           "description":"The Dispatcher component will register with mesos with this secret.",
+           "type":"string",
+           "default":""
+        },
+        "user":{
+           "description":"Tasks will run as this user.",
+           "type":"string",
+           "default":"root"
+        },
+        "log-level":{
+           "type":"string",
+           "description":"log4j log level for Flink cluster.  May be set to any valid log4j log level: https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Level.html",
+           "default":"INFO"
+        },
+        "extra-args":{
+           "description":"Use -D<KEY>=<VALUE> format to provide any additional Flink configuration options.",
+           "type":"string",
+           "default":""
+        },
+        "scala-2-11": {
+          "description":"Use flink compiled for scala-2.11 rather than the default scala-2.10",
+          "type":"boolean",
+          "default":false
+        }
+      }
+    },
+
+    "app-master": {
+      "description": "Properties configuring the Flink AppMaster/JobManager.",
+      "type": "object",
+      "properties": {
+        "cpus": {
+          "default": 1,
+          "description": "CPU shares of AppMaster.",
+          "minimum": 0.1,
+          "type": "number"
+        },
+        "memory": {
+          "default": 1024,
+          "description": "Memory (MB) of AppMaster.",
+          "minimum": 1024,
+          "type": "number"
+        },
+        "heap": {
+          "description": "Heap size (MB) for the AppMaster JVM. This value has to be less or equal than memory.",
+          "type": "number",
+          "default": 256,
+          "minimum": 1
+        }
+      }
+    },
+
+    "task-managers": {
+      "description": "Properties configuring the Flink TaskManagers.",
+      "type": "object",
+      "properties": {
+        "count": {
+          "description": "Number of task managers to spawn.",
+          "type": "integer",
+          "default": 1,
+          "minimum": 1
+        },
+        "cpus": {
+          "description": "CPU shares of each TaskManager.",
+          "type": "number",
+          "default": 1,
+          "minimum": 0.01
+        },
+        "memory": {
+          "description": "Memory (MB) for each task manager.",
+          "type": "number",
+          "default": 1024,
+          "minimum": 256
+        },
+        "heap": {
+          "description": "Heap size (MB) for the TaskManager JVM. This value has to be less or equal than memory.",
+          "type": "number",
+          "default": 512,
+          "minimum": 1
+        },
+        "memory-preallocation": {
+          "description": "Specify whether TaskManager-memory should be allocated when starting up (checked) or when memory is required in the memory manager (unchecked).",
+          "type": "boolean",
+          "default": true
+        }
+      }
+    },
+
+    "high-availability": {
```

Comment:
This is how people install different versions of the same package. They use a different service name. Ok will fix/check config options to comply with that.